style.css

```diff
@@ -0,0 +1,92 @@
+body {
+  font-family: Roboto;
+}
+.wrapper {
+  width: 80%;
+  margin: auto;
+}
+.header {
+  display: flex;
+  justify-content: space-between;
+}
+.header__logo {
+  padding: 40px 0;
+  font-family: PT Serif;
+  font-weight: bold;
+  font-size: 24px;
+  line-height: 31px;
+  position: relative;
+  padding-left: 44px;
+}
+.logo__circle {
+position: absolute;
+left: 0;
+top: 50%;
+transform: translateY(-50%);
+width: 28px;
+height: 28px;
+}
+.logo__circle_color_dark-blue {
+background: #0056ec;
+border-radius: 50%;
+opacity: .7;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/LessonsHTML/pullRequest?prid=6226470)